### PR TITLE
only signed in user can see suggested lines based on grade

### PR DESCRIPTION
**Pull request by nifemioyekunle**
haven't given permissions to see suggested lines, couldn't do it

### CODE REVIEW

File: app/controllers/lines_controller.rb [hunk 1]
```diff
@@ -7,8 +7,15 @@ def index
...
+    authorize(@line)
     @line = Line.find(params[:id])
     @tips = @line.tips
-    authorize(@line)
+  end
+
+  def suggested_lines
+    skip_authorization # pundit to be completed
+    @area = Area.find(params[:area_id])
+    @lines = @area.lines.where(grade: 5.10..5.12)
```
Comment (redvelvet511) [inline app/controllers/lines_controller.rb:18]: Will this range include the letters, like 5.10a, 5.11b?

File: config/routes.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
   resources :areas, only: %i[index] do
     resources :lines, only: %i[index]
+    get "/suggested_lines", to: "lines#suggested_lines"
```
Comment (redvelvet511) [inline config/routes.rb:7]: Should this be with the other GET's below?

File: app/controllers/lines_controller.rb (hunk 1 above)
Comment (nifemioyekunle) [reply app/controllers/lines_controller.rb:18]: yeah

File: config/routes.rb (hunk 2 above)
Comment (nifemioyekunle) [reply config/routes.rb:7]: not sure, but i put it there because its nested in lines  and when i put it with the others it required a /:line_id

**Review by redvelvet511 — APPROVED** [review]

File: app/controllers/lines_controller.rb (hunk 1 above)
Comment (Komezu) [reply app/controllers/lines_controller.rb:18]: I don't think it does...

**Review by Komezu — APPROVED** [review]

**Pull request merged**